.github/workflows/publish.yml

```diff
@@ -12,9 +12,14 @@ jobs:
       - uses: actions/setup-node@v1
         with:
           node-version: 12.x
+          registry-url: https://registry.npmjs.org/
```

Comment:
is this the line that fixes it? The rest seems like it would have no effect except logging

Reply:
yup. it writes a .npmrc ... which npm uses to know where to make requests to.... so dum